### PR TITLE
DOCS-10527 Microsoft Power BI Connector

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+= Microsoft Power BI Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft Power BI (Power BI Connector) synchronizes data and automates business processes between Power BI and third-party applications, either on-premises or in the cloud. Power BI is a suite of analytics tools designed to visualize data from various systems and garner insights.
```
Comment (kbostrom) [inline microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc]:
```suggestion
Anypoint Connector for Microsoft Power BI (Power BI Connector) synchronizes data and automates business processes between Power BI and third-party applications, either on premises or in the cloud. Power BI is a suite of analytics tools designed to visualize data from various systems and garner insights. 
```
```suggestion
Anypoint Connector for Microsoft Power BI (Power BI Connector) synchronizes data and automates business processes between Power BI and third-party applications, either on-premises or in the cloud. Power BI is a suite of analytics tools designed to visualize data from various systems and garner insights. 
```

File: microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+* Credentials to access the target resource
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+Common use cases for Power BI Connector include the ability to automate business processes by integrating Power BI with other IT tools and systems in your organization, including:
+
+* Optimizing sales operations by analyzing your sales data (opportunities, leads, and so on) by feeding data from your CRM system to Power BI, and then using Power BI to visualize patterns and identify trends. 
+* Getting a 360-degree view of your customer by feeding data streams from all customer data stores (CRM, support, marketing, and so on) to Power BI and using Power BI to see a customer-centric view.
```
Comment (kbostrom) [inline microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc]: I wonder if "a 360-degree view" is colloquial?

File: microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* Increasing customer satisfaction and reducing churn risk by notifying the relevant customer account team (using Slack, email, or another messaging system) when customer health drops below the threshold defined in Power BI.
```
Comment (kbostrom) [inline microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc]: ditto "churn risk"?
```suggestion
* Increasing customer satisfaction by notifying the relevant customer account team (using Slack, email, or another messaging system) when customer health drops below the threshold defined in Power BI. 
```
Is there a clearer way to convey "customer health"?

File: microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Common use cases for Power BI Connector include the ability to automate business processes by integrating Power BI with other IT tools and systems in your organization, including:
+
+* Optimizing sales operations by analyzing your sales data (opportunities, leads, and so on) by feeding data from your CRM system to Power BI, and then using Power BI to visualize patterns and identify trends. 
+* Getting a 360-degree view of your customer by feeding data streams from all customer data stores (CRM, support, marketing, and so on) to Power BI and using Power BI to see a customer-centric view.
+* Increasing customer satisfaction and reducing churn risk by notifying the relevant customer account team (using Slack, email, or another messaging system) when customer health drops below the threshold defined in Power BI. 
+
+
+== Authentication Types
+
+Microsoft Power BI Connector uses the OAuth 2.0 authorization code grant connection type, which can be used in apps that are installed on a device to gain access to protected resources.
```
Comment (kbostrom) [inline microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc]: Is this better?
```suggestion
Microsoft Power BI Connector uses the OAuth 2.0 authorization code grant connection type used in apps that are installed on a device to gain access to protected resources.
```
```suggestion
Microsoft Power BI Connector uses the OAuth 2.0 authorization code grant connection type, which can be used in apps that are installed on a device to gain access to protected resources.
```

**Review by kbostrom — DISMISSED** [review]
CE

File: microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (valkyrie69) [reply microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc]: Hhhhmmmmm good question. For now I'll leave it and do a little research. It's mathematical (loosely) so maybe understood.

**Review by sathya0 — APPROVED** [review]

**Pull request merged**